package_io/input_parser.star

```diff
@@ -162,8 +158,7 @@ def default_network_params():
 		"deposit_contract_address":              "0x4242424242424242424242424242424242424242",
 		"seconds_per_slot":                      12,
 		"slots_per_epoch":                       32,
-		"genesis_delay":                         10,
-		"capella_fork_epoch":                    2,
+		"genesis_delay":                         120,
```

Comment:
can you set this to 10?

Reply:
The reason I set it to 120, is to enable CL peering before genesis, so we have a better chance to have healthy peers when genesis hits already.

Reply:
Alrighty; you know best :)